src/ConfigManager.cpp

```diff
@@ -325,13 +343,18 @@ JsonObject ConfigManager::asJson() {
   return obj;
 }
 
+bool ConfigManager::commitChanges() {
+  EEPROM.put(0, magicBytes);
+  return EEPROM.commit();
+}
```

Comment:
@nrwiersma This is newly added. Both storeWifi and writeConfig use this hook to do the final commit, which then ensures the magicByte is set to indicate initialized (prior this was only set on storing wifi)